erp/migrations/0116_auto_20210419_1309.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 3.2 on 2021-04-19 11:09
+
+from django.db import migrations, models
+from erp.models import Accessibilite as Access
+from erp import schema
+
+
+def separate_fields(apps, schema_editor):
+    Accessibilite: Access = apps.get_model("erp", "Accessibilite")
+    for access in Accessibilite.objects.all():
+        if access.cheminement_ext_pente is None:
+            pass
+        elif access.cheminement_ext_pente == schema.PENTE_AUCUNE:
+            access.cheminement_ext_pente = False
+        elif access.cheminement_ext_pente == schema.PENTE_LEGERE:
+            access.cheminement_ext_pente = True
+            access.cheminement_ext_pente_degre_difficulte = schema.PENTE_LEGERE
+        elif access.cheminement_ext_pente == schema.PENTE_IMPORTANTE:
+            access.cheminement_ext_pente = True
+            access.cheminement_ext_pente_degre_difficulte = schema.PENTE_IMPORTANTE
+        else:
+            print(f'Invalid choice for id {access.erp_id} : {access.cheminement_ext_pente}')
```

Comment:
Better raise an exception here so the whole transaction is rolled back (I think? we should check that).

Also could you please ensure formatting python code using `black`? Current CS implies using double-quotes instead of single ones. Thanks!

Reply:
In fact there should be no errors here. I initially set up an error for testing, but there should be no errors. Will revert to raising error.